p/db-busradar-nrw/index.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+// DB Busradar NRW app does not allow selecting specific modes of transport to filter results,
+// so the bitmasks had to be determined by querying some stations and looking at the results..
+const products = [
+	{
+		id: 'nationalExpress',
+		mode: 'train',
+		bitmasks: [1],
+		name: 'InterCityExpress',
+		short: 'ICE',
+		default: true
+	},
+	{
+		id: 'national',
+		mode: 'train',
+		bitmasks: [2],
+		name: 'InterCity & EuroCity',
+		short: 'IC/EC',
+		default: true
+	},
+	// 4: not always used, could not be confirmed..
```

Comment:
Please check if the `4` bit is always being toggled with another one, otherwise mark this comment as resolved. With other endpoints, two bitmasks together represent one product: https://github.com/public-transport/hafas-client/blob/16f98b943eaf58a8c7d57f049f492ef1f49b2c01/p/oebb/products.js#L15

Reply:
I haven't found a 4 on it's own, it seems to appear often on larger stations that have other products as well. There are stations with one of the other products toggled without 4 being toggled as well.
I can't find any hints to the actual meaning of it. If it would be for RegionalExpress, most stations would have wrong data, like Schwelm where there's an 8 but not a 4, while Hagen Hbf correctly also has a 4.
4 also does not seem to be related to something like a tram or subway, as these products do not exist at Hagen Hbf.

Reply:
If you, as an experience local, don't notice any departures/results missing without `4`, that's fine. 😉

Reply:
It seems to only return bus departures in any case, so that wouldn't be a problem. It being returned with different values at different stations may still be interesting though -- subway and tram always(?) return false so these could be removed without any loss of information, unlike this 4..

Reply:
Don't remove anything just because the endpoint doesn't return data for it right now. Maybe the endpoint will have the data in the future.

Reply:
Ok, so should I keep "regionalExp" as is even though we don't know whether that is even correct or can it be renamed to say that it is an unknown mode?

Reply:
- Keep all products if you know their bitmask(s), even if they're not being returned by the endpoint right now.
- Add all products you don't know as e.g. "unknown".